layout/_layout.swig

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="{{ config.language }}">
   <head>
-    {%- include '_partial/head.swig' -%}
+    {%- include '_partial/head.swig' %}
```

Comment:
Thus the `<title>` tag will start in a new line (I think this `\n` is necessary)